src/style-spec/reference/v8.json

```diff
@@ -3764,13 +3755,10 @@
         ]
       },
       "transition": true,
-      "doc": "",
+      "doc": "The fog strength controls the strength of the fog effect. Reducing the fog strength will have the effect of pushing the fog further at the distance, leaving more visible space for haze if enabled.",
```

Comment:
```suggestion
      "doc": "Controls the strength of the fog effect. Reducing the fog strength pushes the fog farther toward the horizon, leaving more visible space for haze, if enabled. When haze energy is zero, this will usually be 1.0.",
```